Dockerfile

```diff
@@ -1,3 +1,16 @@
-FROM starefossen/github-pages:onbuild
+# Get Jekyll build env and Nginx env
+FROM docs/docker.github.io:docs-builder AS builder
 
-CMD jekyll serve -d /_site --watch -H 0.0.0.0 -P 4000
+# Build the docs from this branch
+COPY . /source
+RUN jekyll build --source /source --destination /site
+
+# Reset to bare Nginx image
+FROM nginx:alpine
+
+# Copy the Nginx config
+COPY --from=builder /conf/nginx-overrides.conf /etc/nginx/conf.d/default.conf
```

Comment:
We could use the same as https://github.com/docker/docker.github.io/pull/5366/files#r152847830 here, but up to you :smile: